PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -493,6 +497,7 @@ void AliEmcalJetTask::FillJetConstituents(AliEmcalJet *jet, std::vector<fastjet:
         AliError(Form("Could not find track %d",tid));
         continue;
       }
+      if(fFillConstituents) jet->AddParticleConstituent(t);
```

Comment:
My comments regarding setting embedding still apply here

Reply:
Thanks for notifying! I just overlooked it. Will be fixed in a couple of minutes.

Reply:
Thanks! I'm also wondering about my other comments above regarding size_type

Reply:
I am not sure. To be checked. If it works without compiler error I will add your fix.